pkg/operation/cloudbotanist/localbotanist/controlplane.go

```diff
@@ -73,6 +74,11 @@ func (b *LocalBotanist) GenerateCSIConfig() (map[string]interface{}, error) {
 	return nil, nil
 }
 
+// MetadataServiceAddress returns nil.
+func (b *LocalBotanist) MetadataServiceAddress() (*net.IPNet, error) {
```

Comment:
we are just adding another unusable function here to satisfy the interface, I would rather remove it if possible.

Reply:
What workaround do you suggest?